monitoring/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There are 2-space indents used in this file. Please, follow our code style and use 3 spaces in order to avoid formatting problems in the future.